README.md

```diff
@@ -117,7 +117,7 @@ Change the main branch to the right one, execute the following command:
 Example commands:
 
 ```sh
-  cd my-folder
+  cd my-flder
```

Comment:
- [ ] While you made a great effort to complete this project, I see that you have gone ahead of what is required in this milestone, and this suggests `plagiarism`. However, I cannot prove that to be true. But based on the requirement of this project, the code below is not needed, so you will have to remove it from your schema file and stick to the code for this milestone. 

```sql
- Create owners table
CREATE TABLE owners (
  id SERIAL PRIMARY KEY,
  full_name VARCHAR(255) NOT NULL,
  age INTEGER NOT NULL
);

-- Create species table
CREATE TABLE species (
  id SERIAL PRIMARY KEY,
  name VARCHAR(255) NOT NULL
);

-- Modify animals table
ALTER TABLE animals
  DROP COLUMN species,
  ADD COLUMN species_id INTEGER REFERENCES species(id),
  ADD COLUMN owner_id INTEGER REFERENCES owners(id);


-- Create a table named vets
CREATE TABLE vets (
  id SERIAL PRIMARY KEY,
  name VARCHAR(255),
  age INTEGER,
  date_of_graduation DATE
);


-- Create a "join table" called specializations
CREATE TABLE specializations (
  vet_id INTEGER REFERENCES vets (id),
  species_id INTEGER REFERENCES species (id),
  PRIMARY KEY (vet_id, species_id)
);


-- Create a "join table" called visits
CREATE TABLE visits (
  id SERIAL PRIMARY KEY,
  animal_id INTEGER REFERENCES animals(id),
  vet_id INTEGER REFERENCES vets(id),
  visit_date DATE
);

-- Add an email column to your owners table
ALTER TABLE owners ADD COLUMN email VARCHAR(120);

-- changed visit_date TO date_of_visit
ALTER TABLE visits RENAME COLUMN visit_date TO date_of_visit;
```

Reply:
I'm repeating the block, and I already did the project before, hence the reason why I am ahead.
I am not expected to cancel my progress or restart my project all over, at least not according to the email I was sent by microverse when I was asked to resume.
My project was reviewed and approved yesterday and none of the reviewers asked that I cancel my progress.
Kindly work with today's project as it is present in the code and it literally contains all I was asked to do for today.